stacker/hooks/iam.py

```diff
@@ -30,3 +34,82 @@ def create_ecs_service_role(region, namespace, mappings, parameters,
     conn.put_role_policy("ecsServiceRole", "AmazonEC2ContainerServiceRole",
                          policy.to_json())
     return True
+
+
+def _get_cert_arn_from_response(prefix, response):
+    response_key = '%s_response' % (prefix,)
+    result_key = '%s_result' % (prefix,)
+    result = response[response_key][
+        result_key
+    ]
+    # GET response returns this extra key
+    if 'server_certificate' in result:
+        result = result['server_certificate']
+    return result['server_certificate_metadata']['arn']
+
+
+def ensure_server_cert_exists(region, namespace, mappings, parameters, **kwargs):
+    conn = connect_to_region(region)
+    cert_name = kwargs['cert_name']
+    try:
+        response = conn.get_server_certificate(cert_name)
+        cert_arn = _get_cert_arn_from_response(
+            'get_server_certificate',
+            response,
+        )
+        logger.info('certificate exists: %s (%s)', cert_name, cert_arn)
+    except BotoServerError:
+        upload = raw_input(
+            'Certificate "%s" wasn\'t found. Upload it now? (yes/no) ' % (
+                cert_name,
+            )
+        )
+        if upload != 'yes':
+            return False
+
+        paths = {
```

Comment:
worth maybe making this something you can have in the hook definition?  So you can define the paths to the certs on the local FS?